webclient/app/src/app/commons/appTemplateCard/AppTemplateCard.jsx

```diff
@@ -92,35 +105,45 @@ function AppTemplateCard(props) {
                         </Grid>
                         <Grid item xs={5} align="right">
                             <IconButton onClick={() => handleDialogOpen()}><Edit /></IconButton>
-                            <IconButton onClick={() => handleDelete(appTemplate.Id)}><Delete /></IconButton>
+                            <IconButton onClick={() => setOpenDeleteDialog(true)}><Delete /></IconButton>
                         </Grid>
                     </Grid>
                     <Divider />
-                    <Typography variant="body2" color="text.secondary">
-                        <br />{appTemplate.description}
-                    </Typography>
-                    <List>
-                        <ListItem disablePadding>
-                            <ListItemIcon>
-                                <GitHub />
-                            </ListItemIcon>
-                            <ListItemText primary={appTemplate.location} secondary={t("templateCard.branch") + ": " + appTemplate.branch} />
-                        </ListItem>
-                    </List>
+                    <Grid container spacing={0}>
+                        <Grid item xs={7}>
```

Comment:
```suggestion
                        <Grid item sm={7}>
```
If this is intended please let me know. xs is for most usecases to small